release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -143,6 +143,83 @@ In earlier versions of {product-title}, Ironic container images used {op-system-
 
+[id="ocp-4-12-agent-based-install-asset-input"]
+==== Agent-based installation supports two input modes
+The Agent-based installation supports two input modes:
+
+**Preferred**
+
+* `install-config.yaml` file
+* `agent-config.yaml` file
+
+**Optional**
+
+* Zero Touch Provisioning(ZTP) manifests
+
+With the preferred mode, you can configure the `install-config.yaml` file and specify Agent-based specific settings in the `agent-config.yaml` file.
+For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#about-the-agent-based-installer[About the Agent-based Installer].
+
+[id="ocp-4-12-agent-based-install-fips"]
+==== Agent-based installation supports installing {product-title} cluster in FIPS compliant mode
+Agent-based {product-title} Installer supports {product-title} clusters in Federal Information Processing Standards (FIPS) compliant mode. You must set the value of the `fips` field to `True` in the `install-config.yaml` file.
+For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#agent-installer-configuring-fips-compliance_preparing-to-install-with-agent-based-installer[About FIPS compliance].
+
+[id="ocp-4-12-agent-based-install-disconnected"]
+==== Deploy an Agent-based {product-title} cluster in a disconnected environment
+You can perform an Agent-based installation in a disconnected environment. To create an image that is used in a disconnected environment, the `imageContentSources` section in the ` install-config.yaml` must contain the mirror information or `registries.conf` file if you are using ZTP manifests. The actual configuration settings to use in these files are supplied by either the `oc adm release mirror` or `oc mirror` command.
+For more information, see xref:../installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.html[Understanding disconnected installation mirroring].
+
+[id="ocp-4-12-agent-based-install-dual-stack"]
+==== Agent-based installation supports single and dual stack networking
+You can create the agent ISO image with the following IP address configurations:
+
+* IPv4
+* IPv6
+* IPv4 and IPv6 in parallel (dual-stack)
+
+For more information, see xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent_installing-with-agent-based-installer[Dual and single IP stack clusters].
+
+[id="ocp-4-12-agent-based-install-mce"]
+
+==== The agent deployed {product-title} cluster can be used as a hub cluster
+
+You can install the multicluster engine for Kubernetes(MCE) Operator and deploy a hub cluster with the Agent-based Installer.
+
+// TODO doc PR not merged see xref:..
+
+[id="ocp-4-12-agent-based-install-validations"]
+==== Agent-based installation performs installation validations
+
+The Agent-based Installer performs validations on:
```

Comment:
IBM SG suggests avoiding leading a sentence into a list. Maybe: "... performs validations on the following:"?